docs/index.rst

```diff
@@ -479,6 +479,18 @@ features <https://docs.confluent.io/current/security.html>`__:
 
 There is an OpenLDAP server running in the demo, and each Kafka broker in the demo is configured with MDS and can talk to LDAP so that it can authenticate clients and Confluent Platform services and clients.
 
+Zookeeper has two listener ports:
+
++---------------+----------------+--------------------------------------------------------------------+--------+--------+
+| Name          | Protocol       | In this demo, used for ...                                         | zookeeper       |
++===============+================+====================================================================+=================+
+| N/A           | SASL/DIGEST-MD5| Validating of Trial License (no TLS support) for Rest and SR       | 2181            |
```

Comment:
```suggestion
| N/A           | SASL/DIGEST-MD5| Validating trial license for |crest| and |sr| (no TLS support)      | 2181            |
```

Reply:
@sdanduConf 

1. Are you aware of a docs page that documents no TLS support for validating license?
2. Is it just trial license or or non-trial license too?  If latter, should we remove the word `trial` from the table?

Reply:
I believe it is just trial licenses.  @skaundinya15 could you perhaps comment since I believe you removed the ZK license code?

Reply:
@ybyzek : I see release note here -http://staging-docs.confluent.io.s3-website-us-west-2.amazonaws.com/PR/4071/5.5.0-SNAPSHOT/release-notes/index.html#schema-registry.

Reply:
@rondagostino The ZK licensing code that I removed was for the trial license that used ZK instead of the `LicenseManager`. If this is for `5.5.x` there are still components that use ZK for verifying trial licenses, but for future versions ZK based trial licensing will be gone.